src/components/tab/__tests__/Tabs.Test.tsx

```diff
@@ -46,17 +46,17 @@ describe('<Tabs />', () => {
     expect(screen.getByText('Eins')).toBeInTheDocument();
     expect(screen.queryByText('Zwei')).not.toBeInTheDocument();
   });
-  it('changes tab on click', async () => {
```

Comment:
I did some tests and forgot to remove it

Reply:
Thank you!